File: frontend/henn-laszlo/projects/invitation-data-access/src/lib/models/admin-invitation-details.ts
export interface AdminInvitationDetails {
  readonly id: string;
  readonly titleHu: string;
  readonly titleEn: string | null;
  readonly year: number | null;
  readonly altTextHu: string | null;
  readonly altTextEn: string | null;
  readonly hasImage: boolean;
  readonly isPublished: boolean;
  readonly displayOrder: number;
}

File: backend/src/Modules/Modules.Invitation.Application/Invitations/GetAdminById/AdminInvitationDetails.cs
namespace Modules.Invitation.Application.Invitations.GetAdminById;

public sealed record AdminInvitationDetails(
    Guid Id,
    string TitleHu,
    string? TitleEn,
    int? Year,
    string? AltTextHu,
    string? AltTextEn,
    bool HasImage,
    bool IsPublished,
    int DisplayOrder);